app/services/websites/create.rb

```diff
@@ -0,0 +1,56 @@
+module Websites
+  class Create
+    # From the controller, use it like this:
+    #
+    # Websites::Create.call(params, user)
+    def self.call(params, user)
+      new(params, user).call
+    end
+
+    def initialize(params, user)
+      @params = params
+      @user = user
+      @website = Website.where(url: params[:website][:url])
+    end
+
+    def call
+      @descript = create_website
+      update_statistics
+      find_or_create_user
+      fetch_rank
+      @descript
+    end
```

Comment:
Why is the variable called `@descript`? Also, is it required to be an instance variable, or can we make it a local var?